business/proxy_status.go

```diff
@@ -74,3 +76,47 @@ func xdsStatus(sent, acked string) string {
 	// Since the Nonce changes to uuid, so there is no more any time diff info
 	return "Stale"
 }
+
+func (in *ProxyStatus) GetConfigDump(namespace, pod string) (interface{}, error) {
+	var err error
+	promtimer := internalmetrics.GetGoFunctionMetric("business", "ProxyStatus", "GetConfigDump")
+	defer promtimer.ObserveNow(&err)
+
+	dump, err := in.k8s.GetConfigDump(namespace, pod)
+	if err != nil {
+		return nil, err
+	}
+
+	cd := models.NewConfigDump(dump)
```

Comment:
Discussed with @xeviknal if at this point, as response is received in JSON format, no really need to bring the protobuf dependencies that are being pulled from Envoy.
Perhaps in the context of this feature we could do something similar as IstioConfig and use a simple struct model with some interface{} for inner fields.